### ISSUE TITLE
Relative file path note creation

### ISSUE THREAD

**Issue by TheSquake** [SimonTC/obsidian-note-autocreation#9]
Hello,

When I have a folder structure like

```
note.md
Folder1\
    Folder2\
        note2.md
```

and in note2 I would like to make a link to a new note @./Folder3/note3.md
I would like to see this happen 

```
note.md
Folder1\
    Folder2\
        note2.md
        Folder3\
             note3.md
```

instead of

```
note.md
Folder1\
    Folder2\
        note2.md
Folder3\
     note3.md
```

this behavior happens for me both with regular [[]] syntax and @ symbol from your extension. Do you have plan implementing relative path note creation like this ?

**Comment by SimonTC** [comment]
Hi,

Thanks a lot for the request. It is a really good idea and something I actually also could use myself. I will add it to my list of features to work on.

**Comment by SimonTC** [comment]
Hi @TheSquake . 

This feature is now included in release 1.3.0.

**Comment by TheSquake** [comment]
Amazing ! this will make my life so much easier now 💓
